File: GitToolBox/src/main/java/zielu/gittoolbox/help/GithubWikiHelpProvider.java
package zielu.gittoolbox.help;

import com.intellij.openapi.help.WebHelpProvider;
import org.jetbrains.annotations.NotNull;
import org.jetbrains.annotations.Nullable;

public class GithubWikiHelpProvider extends WebHelpProvider {
  @Nullable
  @Override
  public String getHelpPageUrl(@NotNull String helpTopicId) {
    String anchor = HelpKey.findKeyById(helpTopicId)
                        .map(HelpKey::getAnchorId)
                        .map(anchorId -> "#" + anchorId)
                        .orElse("");
    return "https://github.com/zielu/GitToolBox/wiki/Manual" + anchor;
  }
}

File: src/main/kotlin/zielu/gittoolbox/help/GithubWikiHelpProvider.kt
package zielu.gittoolbox.help

import com.intellij.openapi.help.WebHelpProvider

internal class GithubWikiHelpProvider : WebHelpProvider() {

  override fun getHelpPageUrl(helpTopicId: String): String {
    val anchor = HelpKey.findKeyById(helpTopicId)?.anchorId?.let { "#$it" } ?: ""
    return "https://github.com/zielu/GitToolBox/wiki/Manual$anchor"
  }
}
